### ISSUE TITLE
The app shows count is 0 while loading firebase.

### ISSUE THREAD

**Issue by kasperpeulen** [Janamou/firebase-counter-steps#3]
The app shows count is 0 while loading firebase.

Would be useful to see a workaround for that. For example, you could show a `Loading...` text until the count is initialized. If you do so, I think the code for updating could also be simplified to:

```dart 
  like() {
    ref.transaction((count) => count + 1);
  }

  dislike() {
    ref.transaction((count) => count - 1);
  }
```

**Comment by Janamou** [comment]
@kasperpeulen Yes, I know. The codelab had to be as much simple as possible and only for 20 minutes or less (lot's of things were deleted...). Unfortunately there was no space for "make it even fancier" stuff.

The codelab is incremental and it has to start with the simplest thing possible - e.g. implement only a little code to show something - a component with 0 value (it's step 5 of the codelab) and see it in the web browser (without the 0 value, nothing would be rendered).

Then the component is updated to the actual value from Firebase. But I think that instructing people to remove the default value of 0 could take some time (e.g. explain them why etc.) and could be potential source of errors and misunderstanding. So the value of 0 stayed there...

**Comment by kasperpeulen** [comment]
@Janamou I see, I just found out about this codelab, but I see it was for IO now. I like this codelab a lot, just thought doing something like a loading screen until the count is could make it better. But this codelab is probably frozen now, so nvm :)
